localtypings/pxtarget.d.ts

```diff
@@ -551,6 +551,7 @@ declare namespace pxt {
         usedBlocks: Map<number>;
         snippetBlocks: Map<Map<number>>;
         highlightBlocks: Map<Map<number>>;
+        validateBlocks: Map<Map<string[]>>;
```

Comment:
Not in this pr, but we'll want to solve the issue of having a block with `// @highlight @validate-exists` 'counting twice' eventually as it would feel buggy for that to require the same block twice. One way we discussed in a sync earlier would be to make this a `Map<Map<BlockData[]>>` , where `BlockData` would be something like

```ts
interface BlockData {
    type: string; // the pxtc blockId that is currently stored in validateBlocks
    id: string; // the id attached to the block in the blockly workspace
}
```

e.g. in https://github.com/microsoft/pxt/pull/9675/files#diff-cae476038f191f5305fbc82c2c9f96c74216375046a125e25dafe2bb12be86b1R119 `id` would come from `blk.id`. The ID is just a random value generated when going from xml -> blockly workspace but can be useful to e.g. identify when the tags came from separate (similar) blocks